include/mgp_mock.py

```diff
@@ -0,0 +1,1011 @@
+import datetime
+import inspect
+import sys
+import typing
+from collections import namedtuple
+from functools import wraps
+
+import _mgp_mock
+import kafka
+import networkx as nx
+import pulsar
+
```

Comment:
Add some high level docs similar to what is in mgp.py.

Reply:
Added